pkg/services/ngalert/notifier/encryption.go

```diff
@@ -0,0 +1,108 @@
+package notifier
+
+import (
+	"context"
+	"encoding/base64"
+	"errors"
+	"fmt"
+
+	"github.com/grafana/grafana/pkg/services/ngalert/api/tooling/definitions"
+	"github.com/grafana/grafana/pkg/services/ngalert/models"
+	"github.com/grafana/grafana/pkg/services/ngalert/store"
+	"github.com/grafana/grafana/pkg/services/secrets"
+)
+
+type Encryption interface {
```

Comment:
```suggestion
type Crypto interface {
```

Maybe even `Secrets` so that you can reference it as `notifier.Secrets` or `notifier.Crypto` because that aligns with the rest of Grafana. Encryption is def not correct because this also decrypts.